express/properties/workflow.py

```diff
@@ -1,6 +1,178 @@
 from express.properties import BaseProperty
+import os
+import copy
+from typing import List
 
 
+# Note for naming convention: code uses snake_case, JSON uses camelCase
+
+
+# Todo: This is the quick implementation of ExabyteML, and will be depreciated eventually
+class MLQuickImplementation(BaseProperty):
+    """
+    Quick implementation of the new version of ExabyteML
```

Comment:
1. This doesn't explain anything - we need to actually tell what's going on
2. Let's add an example of the workflow structure that we expect here somewhere to understand where you get the unit names etc. from`head-set-predict-status` - they just appear out of nothing right now

Reply:
1. Updated the docstring with a lot more information
2. Added a detailed explanation of the workflow structure, including the two specific flowchartID's we reference later on in the class